_docs/sysadmin/installation.md

```diff
@@ -26,23 +26,23 @@ _Note: These instructions should be run under root/sudo._
 1. [Install Ubuntu 18.04 server edition (or other supported distro)](server_os)
 
 
-2. After installing the operating system, clone the git repository:
-
+2. Run the bootstrap script:
    ```
-   mkdir -p /usr/local/submitty/GIT_CHECKOUT
-   git clone https://github.com/Submitty/Submitty.git /usr/local/submitty/GIT_CHECKOUT/Submitty
+   curl https://raw.githubusercontent.com/Submitty/Submitty/bootstrap/.setup/bootstrap.sh | bash
    ```
-
-3. Run the automated portion of the install.
+   
+   or clone the git repository and run the installer (requires git and lsb-release to be installed):
 
    ```
+   mkdir -p /usr/local/submitty/GIT_CHECKOUT
+   git clone https://github.com/Submitty/Submitty.git /usr/local/submitty/GIT_CHECKOUT/Submitty
    cd /usr/local/submitty/GIT_CHECKOUT/Submitty
    bash ./.setup/install_system.sh
    ```
 
-   You will be asked several questions by the
-   [CONFIGURE_SUBMITTY.sh script](https://github.com/Submitty/Submitty/blob/master/.setup/CONFIGURE_SUBMITTY.sh).
-   These questions are:
+   Note: During installation, you will be asked several questions by the
+   [CONFIGURE_SUBMITTY.py script](https://github.com/Submitty/Submitty/blob/master/.setup/CONFIGURE_SUBMITTY.py)
+   script. These questions are:
```

Comment:
There are two `script`s here. It may be better if one of them is removed.

> Note: During installation, you will be asked several questions by the [CONFIGURE_SUBMITTY.py script](https://github.com/Submitty/Submitty/blob/master/.setup/CONFIGURE_SUBMITTY.py) script.

Reply:
Thanks for pointing this out. Fixed.